rein/cli.py

```diff
@@ -1185,87 +1174,46 @@ def flash_errors(form):
                        error
                        ))
 
-    @app.route("/setup", methods=['POST', 'GET'])
-    def start_setup():
-        log = rein.get_log()
-        form = SetupForm(request.form)
-
-        from rein.lib.user import User
-
-        if request.method == 'POST' and form.validate_on_submit():
-            new_identity = User(form.name.data,
-                                form.contact.data,
-                                form.maddr.data,
-                                form.daddr.data,
-                                form.dkey.data,
-                                form.will_mediate.data,
-                                form.mediator_fee.data,
-                                rein.testnet)
-            rein.session.add(new_identity)
-            rein.session.commit()
-            data = {'name': form.name.data,
-                    'contact': form.contact.data,
-                    'maddr': form.maddr.data,
-                    'daddr': form.daddr.data,
-                    'dkey': form.dkey.data,
-                    'will_mediate': form.will_mediate.data,
-                    'mediator_fee': form.mediator_fee.data,
-                    'testnet': rein.testnet}
-            filename = rein.backup_filename + '-' + ''.join(random.SystemRandom().choice(string.ascii_uppercase + string.digits) for _ in range(6))
-            if not os.path.isfile(filename):
-                f = open(filename, 'w')
-                try:
-                    f.write(json.dumps(data))
-                    click.echo("Backup saved successfully to %s" % filename)
-                except:
-                    RuntimeError('Problem writing user details to json backup file.')
-                    return render_template("setup.html",
-                                           form=form)
-                f.close()
+    @app.route('/setup')
+    def setup():
```

Comment:
Found an issue getting past setup. This function name conflicts silently with the setup argument to the `rein start` command. Please update the function name and this issue should be resolved.